debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -1008,6 +1008,51 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
                     "for data change, schema change, transaction, heartbeat event etc.")
             .withDefault(DefaultTopicNamingStrategy.class.getName());
 
+    public static final Field MONGODB_MULTI_TASK_ENABLED = Field.create("mongodb.multi.task.enabled")
```

Comment:
Do we really need to have such a config option? IMHO just the setting of `tasks.max > 1` should trigger the behaviour.
But as a safety concern (given we are going to merge it to stable branch) I'd keep this yet use `Field.createInternal`. It would be `false` by default in 3.0 and `true` in 3.1. The option could be then removed completely.

Reply:
I would definitively keep an option that specifically enables **THIS TYPE** of multitask As in the future we might want to do multi tasking based on e.g. databases.

Reply:
Agree with @jcechace on this :) There is also an existing replset based multi-tasking for Mongo, so we wanted to be very explicit about having a flag to turn the feature on/off.

Reply:
@ning-stripe RS-based one was removed as it was highly problematic